common/federation/src/test/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProviderSmokeTest.java

```diff
@@ -29,4 +31,32 @@ void testCanLoadKeystore() {
     assertThat(cut.createFeignClient()).isNotNull();
   }
 
+  @Test
+  void testCertificate(){
+    Ssl ssl = new Ssl();
+    ssl.setCertificateDn("X.509");
+    assertThat(ssl.getCertificateDn(),is("X.509"));
+  }
+
+  @Test
+  void testCertificateSHA(){
+    Ssl ssl = new Ssl();
+    ssl.setCertificateSha("randomString123123");
+    assertThat(ssl.getCertificateSha(),is("randomString123123"));
+  }
+
+  @Test
+  void testGetBaseUrl(){
+    Ssl ssl = new Ssl();
+    ssl.setBaseUrl("https://github.com/cwa-server/tree/main");
+    assertThat(ssl.getBaseUrl().isNotNull());
```

Comment:
You have the closing paranthenses at the wrong place here; it should be `assertThat(ssl.getBaseUrl()).isNotNull()`